docs/docs/how-code-splitting-works.md

```diff
@@ -12,9 +12,8 @@ During the [Write Out Pages](/docs/write-pages/#async-requiresjs) bootstrap phas
 
 ```javascript
 exports.components = {
-  "component--src-blog-js": () =>
-    import("/home/site/src/blog.js"),
-    /* webpackChunkName: "component---src-blog-js" */
+  "component--src-blog-js": () => import("/home/site/src/blog.js"),
+  /* webpackChunkName: "component---src-blog-js" */
```

Comment:
I think this is a pre-existing typo, the `webpackChunkName` comment should be part of the import call:

```diff
-  "component--src-blog-js": () => import("/home/site/src/blog.js"),
-   /* webpackChunkName: "component---src-blog-js" */
+  "component--src-blog-js": () => import("/home/site/src/blog.js" /* webpackChunkName: "component---src-blog-js" */),
```